governance/program/src/processor/process_withdraw_governing_tokens.rs

```diff
@@ -67,15 +74,36 @@ pub fn process_withdraw_governing_tokens(
 
     token_owner_record_data.assert_can_withdraw_governing_tokens()?;
 
-    transfer_spl_tokens_signed(
-        governing_token_holding_info,
-        governing_token_destination_info,
-        realm_info,
-        &get_realm_address_seeds(&realm_data.name),
-        program_id,
-        token_owner_record_data.governing_token_deposit_amount,
-        spl_token_info,
-    )?;
+    match expected_mint_info {
+        Some(mint_info) => {
+            let additional_accounts = account_info_iter.as_slice();
```

Comment:
nit (same as above): would move that up

Reply:
this is only used in the case of token extensions, since it requires a mint in the transfer_checked(). not sure if it is necessary to move up.